docs/src/lang/records.md

```diff
@@ -69,9 +69,6 @@ and `[type |-> "B", b |-> TRUE]` have the joint type:
 
 ## Operators
 
-In the Python examples, we are using the package [frozendict], to produce an
```

Comment:
I think we should still use frozendict where appropriate. And it is needed in parts of this section.